packages/devextreme/scss/widgets/fluent/_colors.scss

```diff
@@ -238,6 +245,8 @@ $scrollview-pulldown-path: null !default;
   $base-success-selected: rgba(11, 81, 11, 1) !default; // #0B510B
 
   $scrollview-pulldown-path: data-uri('images/widgets/fluent/color-schemes/light/pulldown.png') !default;
+
+  $base-color-disabled: rgba(189, 189, 189, 1) !default; // #BDBDBD
```

Comment:
I'm not sure it's ok we specify a new variable on the single theme.
@EugeniyKiyashko  check plz

Reply:
We have 'base-disabled-color' in our variables with value `e0e0e0` in light theme. In your case using colors '#BDBDBD', '#5C5C5C' and there is `$base-foreground-disabled` with this values. Please, use `$base-foreground-disabled` directly and let's remove `$base-color-disabled` variable and I would prefer not to publish it.

Reply:
[[refactor(fluent): Remove extra variable && Remove ways to publish variables](07cb4534d39dda5d2d3359df2bf5fe85dd3cd4ad)]

Reply:
[[refactor(fluent): Remove extra variable && Remove ways to publish variables](154c4f1fffced1986890ef03376011e70927dd60)]